CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Changed
 - Contributions Catalog login callback endpoint to /catalog/auth/callback. [#803](https://github.com/rokwire/rokwire-building-blocks-api/issues/803) 
+- Removed unused method in token utility. [#837](https://github.com/rokwire/rokwire-building-blocks-api/issues/837) 
```

Comment:
Similarly, please move this under the Security section too. Thanks.

Reply:
Or you can wait until the other PR #836  gets merged.

Reply:
let's wait until it gets merged because it will generate merge conflict... :)

Reply:
I have merged that PR. Thanks.

Reply:
changed